docs/quick-ingestion-guides/looker/setup.md

```diff
@@ -0,0 +1,145 @@
+---
+title: Setup
+---
+
+# Looker Ingestion Guide: Setup & Prerequisites
+
+In order to configure ingestion from Looker, you'll first have to ensure you have an API key to access the Looker resources.
+
+## Looker Prerequisites
+Follow below steps to create API key.
+
+1. **Login:** Login to `https://<instance-name>.cloud.looker.com`. Replace `<instance-name>` by your looker instance name. For example: `https://abc.cloud.looker.com`
+
+2. **Admin Panel:** Navigate to `Admin Panel` on looker home page.
+
+      <p align="center">
+   <img width="75%" alt="Looker home page" src="http://raw.githubusercontent.com/datahub-project/static-assets/main/imgs/guides/looker/looker-home-page.png"/>
+      </p>
+
+3. **Roles Panel:** Search for `Roles` on `Admin Panel` and click `Roles` to open `Roles Panel`.
+
+      <p align="center">
+   <img width="75%" alt="Looker roles search" src="http://raw.githubusercontent.com/datahub-project/static-assets/main/imgs/guides/looker/looker-roles-search.png"/>
+      </p>
+
+4. **Create A New Permission Set:** On `Role Panel` follow below steps to create a `New Permission Set`.
+
+   a. Go to `New Permission Set`
+
+      <p align="center">
+      <img width="75%" alt="Looker new permission set" src="https://raw.githubusercontent.com/datahub-project/static-assets/main/imgs/guides/looker/looker-new-permission-set-button.png"/>
+      </p>
+
+   b. Set name for  `New Permission Set`, says `DataHub Connector Permission Set` and select following permissions
+
+      - access_data
```

Comment:
also here, the line spacing should be narrower 
or you could utilize <details></details> to make it collapsible 

<img width="665" alt="image" src="https://github.com/datahub-project/datahub/assets/26561086/9a0f6e06-ef36-4091-953d-194f67734335">

Reply:
done